### PR TITLE
task: add otlp-http port and instructions for lighting up console

### PR DESCRIPTION
**Pull request by o11yguru**
### Any background context you want to provide for reviewers?
* Add `otlp-http` port
* Add curl and grpcurl requests and sample payloads
* Update console helm chart version
* Remove unused variable name

### CODE REVIEW

**Review by arcanez — APPROVED** [review]

File: templates/mdai-operator.yaml [hunk 1]
```diff
@@ -60,6 +60,8 @@ spec:
               protocol: TCP
             - name: otlp-grpc
               port: 4317
+            - name: otlp-http
```
Comment (s0b01ev) [inline templates/mdai-operator.yaml:63]: as discussed, this ports section is not needed, if standard receivers used. all those ports will end up in the Service/Ingress without mentioning them here

File: Makefile [hunk 2]
```diff
@@ -45,6 +45,11 @@ config-aws: dot-env
 config-mdai:
 	envsubst < templates/mdai-operator.yaml > lib/mdai-operator.yaml
 
+.PHONY: config-otel
+.SILENT: config-otel
+config-otel:
+	envsubst < templates/otel-tmpl.yaml > lib/otel-operator.yaml
+
```
Comment (arcanez) [inline Makefile:52]:
```suggestion
```

File: Makefile [hunk 3]
```diff
@@ -71,7 +76,7 @@ npm-build:
 
 .PHONY: config
 .SILENT: config
-config: go-mod config-aws config-mdai
+config: go-mod config-aws config-mdai config-otel
```
Comment (arcanez) [inline Makefile:79]:
```suggestion
config: go-mod config-aws config-mdai
```

File: templates/otel-tmpl.yaml [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+      alb.ingress.kubernetes.io/target-type: ip
+      kubernetes.io/ingress.class: alb
+      alb.ingress.kubernetes.io/certificate-arn: $MDAI_UI_ACM_ARN
+      alb.ingress.kubernetes.io/listen-ports: '[{"HTTPS": 443}]'
+    collectorEndpoints:
+      otlp: otlp.grpc.endpoint.collector.your-domain.io
+      jaeger: jaeger.grpc.endpoint.collector.your-domain.io
+    ingressClassName: alb
+    ruleType: path
+    type: aws
```
Comment (arcanez) [inline templates/otel-tmpl.yaml:25]:
```suggestion
```
not needed

**Review by arcanez — CHANGES_REQUESTED** [review]

File: lib/decisive-engine-aws-cdk-stack.ts [hunk 5]
```diff
@@ -236,8 +236,16 @@ export class DecisiveEngineAwsCdkStack extends cdk.Stack {
...
+    const mdaiOperatorCrManifestYaml = yaml.load(readFileSync(path.join(__dirname, 'mdai-operator.yaml'), { encoding: 'utf-8' })) as Record<string, any>;
+    const mdaiOperatorCrManifest = cluster.addManifest('mdaiOperatorCrManifest', mdaiOperatorCrManifestYaml);
+    mdaiOperatorCrManifest.node.addDependency(mdaiOperator);
+
+    const otelOperatorCrManifestYaml = yaml.load(readFileSync(path.join(__dirname, 'otel-operator.yaml'), { encoding: 'utf-8' })) as Record<string, any>;
+    new eks.KubernetesManifest(this, 'otelOperatorCrManifest', {
+      cluster,
+      manifest: [otelOperatorCrManifestYaml],
+      overwrite: true,
+    }).node.addDependency(mdaiOperatorCrManifest);
```
Comment (arcanez) [inline lib/decisive-engine-aws-cdk-stack.ts:248]:
```suggestion
    const mdaiOperatorCrManifest = yaml.load(readFileSync(path.join(__dirname, 'mdai-operator.yaml'), { encoding: 'utf-8' })) as Record<string, any>;
    cluster.addManifest('mdaiOperatorCrManifest', mdaiOperatorCrManifest).node.addDependency(mdaiOperator);
```

operator applies annotations so applying the otel operator manifest is not needed.

File: test/decisive-engine-aws-cdk.test.ts [hunk 6]
```diff
@@ -89,7 +89,11 @@ describe('DecisiveEngine', () => {
                     Namespace: 'default',
                     CreateNamespace: true,
                     Release: 'mdai-console',
+<<<<<<< HEAD
                     Version: '0.2.2',
+=======
+                    Version: '0.2.1',
+>>>>>>> d1aa084 (task: bump version of console to 2.1)
```
Comment (arcanez) [inline test/decisive-engine-aws-cdk.test.ts:96]:
```suggestion
                    Version: '0.2.2',
```

File: templates/otel-tmpl.yaml [hunk 7]
```diff
@@ -0,0 +1 @@
+  mode: deployment
```
Comment (arcanez) [inline templates/otel-tmpl.yaml:1]: remove this file?

**Review by arcanez — CHANGES_REQUESTED** [review]

File: Makefile [hunk 8]
```diff
@@ -69,6 +69,11 @@ npm-init:
 npm-build:
 	npm run build;
 
+# FAKING THIS SO IT DOESN"T FEEL LIKE MULTIPLE RUNS OF THE SAME COMMAND
+.PHONY: init
+.SILENT: init
+init: go-mod config-aws config-mdai config-otel
```
Comment (arcanez) [inline Makefile:75]:
```suggestion
init: go-mod config-aws config-mdai
```

Comment (s0b01ev) [reply Makefile:75]: What do we need this target for? No needs to run same command multiple times. With the current flow it's just 

$ make cert
$ make config
$ make install

Am I missing something here?

**Review by arcanez — APPROVED** [review]

**Pull request merged**